src/components/StatePicker.js

```diff
@@ -15,30 +16,57 @@ const propTypes = {
     label: PropTypes.string,
 
     /** A callback method that is called when the value changes and it received the selected value as an argument */
```

Comment:
```suggestion
    /** A callback method that is called when the value changes and it receives the selected value as an argument. */
```

Reply:
Thanks for the suggestion, now its updated